libavfilter/dnn/dnn_backend_openvino.c

```diff
@@ -31,6 +31,9 @@
 #include "libavutil/opt.h"
 #include "libavutil/avstring.h"
 #include "libavutil/detection_bbox.h"
+#include "libavutil/hwcontext.h"
+#include "libavutil/hwcontext_vaapi.h"
+#include <va/va.h>
```

Comment:
`#include "libavutil/hwcontext_vaapi.h"` has` include "va/va.h"`, so this line maybe redundant.

Reply:
BTW it is available only under CONFIG_VAAPI

Reply:
> `#include "libavutil/hwcontext_vaapi.h"` has` include "va/va.h"`, so this line maybe redundant.

Thx for review, I will remove it.

> BTW it is available only under CONFIG_VAAPI

Ok, I will use `#if CONFIG_VAAPI` in related include statement and functions.